source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -0,0 +1,65 @@
+title: Check the location of your ``.so`` files
+ref: check-location
+level: 4
+content: |
+  The |odbc-driver-name| files are named ``libmdbodbca.so`` and
+  ``libmdbodbcw.so``. After downloading the |odbc-driver-name|,
+  move them to your ``/usr/local/lib`` directory.
+---
+title: Create an ``odbc.ini`` file
+ref: create-odbc-ini
+level: 4
+content: |
+  The file ``/etc/odbc.ini`` defines your Data Source Names. Below is
+  an example ``odbc.ini`` file which is configured to use the
+  |odbc-driver-name| with a DSN called ``MongoDBODBC``. Replace all
+  necessary placeholder values with values for your system.
+
+  The file named ``libmdbodbca.so`` is the ANSI driver, and the
+  file named ``libmdbodbcw.so`` is the Unicode driver. Specify
+  the desired driver in your ``odbc.ini`` file.
+
+  .. include:: /includes/fact-ansi-unicode-driver.rst
+
+  .. code-block:: none
+
+     [MongoDBODBC]
+     Description = ODBC for MongoDB BI Connector
+     Driver      = /usr/local/lib/<libmdbodbca.so|libmdbodbcw.so>
+     Trace       = Off
+     TraceFile   = stderr
+     ReadOnly    = yes
+     SERVER      = <server address>
+     PORT        = 3307
+     USER        = <username>
+     PASSWORD    = <password>
+     DATABASE    = <dbname>
+---
+title: Install unixODBC
+ref: install-unixodbc
+level: 4
+content: |
+  Install ``unixODBC`` with your preferred package manager.
+  Alternatively, you can download binaries from the `unixODBC website
+  <http://www.unixodbc.org>`_.
+---
+title: Test with isql
+ref: test-with-isql
+level: 4
+content: |
+  The ``unixODBC`` package includes an SQL client called ``isql``. You
+  can use it to test your DSN.
+
+  Ensure that |bi-short| is running at the address specified in your
+  ``odbc.ini`` file and start ``isql`` from the command prompt,
+  specifying your DSN as a parameter.
```

Comment:
Consider including a better segue to the example:
"The following example uses ``isql`` to connect to the ``MongoDBODBC`` DSN:"

Reply:
Done.